ci/owncloud/owncloud_definitions.sh

```diff
@@ -105,6 +105,33 @@ browse_url() {
     echo "$browse_url"
 }
 
+download_url() {
+    # Calculate and echo the download url for a file
+    #
+    # $1 relative path to the file from the base_url
+    # $2 base_url
+    local download_url
+
+    download_url="$(browse_url "$(dirname "$1")" "$2"/download)&files=$(basename "$1")"
```

Comment:
I believe this should be
```suggestion
    download_url="$(browse_url "${1%%/*}" "$2"/download)&files=${1#*/}"
```
i.e. break off the first component of the path only, not the last component.